.github/ISSUE_TEMPLATE.md

```diff
@@ -1,19 +1,24 @@
-**Rancher Versions:**
-Server:
+**Rancher versions:**
+rancher/server:
+rancher/agent:
```

Comment:
We don't actually need rancher/agent as it is tied to each rancher/server release. When you upgrade this should be auto upgraded as well.